pkg/providers/amifamily/bottlerocket.go

```diff
@@ -141,5 +141,6 @@ func (b Bottlerocket) FeatureFlags() FeatureFlags {
 		UsesENILimitedMemoryOverhead: false,
 		PodsPerCoreEnabled:           false,
 		EvictionSoftEnabled:          false,
+		SupportsENILimitedPodDensity: true,
```

Comment:
nit: I'm not sure that this is needed if it uses the default value

Reply:
Does Bottlerocket support ENILimitedPodDensity?